src/Raven.TestDriver/RavenTestDriver.cs

```diff
@@ -88,6 +88,8 @@ protected IDocumentStore GetDocumentStore(GetDocumentStoreOptions options = null
 
             PreInitialize(store);
 
+            store.Conventions.DisableTopologyCache = true;
```

Comment:
let's set it up few lines up in place where are we creating a document store. This way you are allowing user to turn it on.

Reply:
done.